pkg/test/framework/features/features.yaml

```diff
@@ -61,15 +61,51 @@ features:
     user:
     ingress:
       mtls:
+        server-keyrotation:
+        generic-compoundrotation:
+        nongeneric-compoundrotation:
+        gateway:
+          invalid-secret:
+          valid-secret:
+          K8sca:
       tls:
+        secretrotation:
+        gateway:
+          invalid-secret:
+          K8sca:
     egress:
       mtls:
         sds:
       tls:
         filebased:
         sds:
+    vault:
+      sds:
```

Comment:
@myidpt is vault.sds a proper feature label?

Reply:
Nope, I don't think we need to test this.

Reply:
ok i will remove this feature